device/arista/x86_64-arista_7800r3_48cq2_lc/Arista-7800R3-48CQ2-C48/port_config.ini

```diff
@@ -1,51 +1,51 @@
+Ethernet80          46,47     Ethernet21/1        21     Ext   100000    0          21             8
+Ethernet84          42,43     Ethernet22/1        22     Ext   100000    0          22             8
+Ethernet88          44,45     Ethernet23/1        23     Ext   100000    0          23             8
+Ethernet92          40,41     Ethernet24/1        24     Ext   100000    0          24             8
+Ethernet96          94,95     Ethernet25/1        25     Ext   100000    1          25             8
+Ethernet100         90,91     Ethernet26/1        26     Ext   100000    1          26             8
+Ethernet104         92,93     Ethernet27/1        27     Ext   100000    1          27             8
+Ethernet108         88,89     Ethernet28/1        28     Ext   100000    1          28             8
+Ethernet112         86,87     Ethernet29/1        29     Ext   100000    1          29             8
+Ethernet116         82,83     Ethernet30/1        30     Ext   100000    1          30             8
+Ethernet120         84,85     Ethernet31/1        31     Ext   100000    1          31             8
+Ethernet124         80,81     Ethernet32/1        32     Ext   100000    1          32             8
+Ethernet128         78,79     Ethernet33/1        33     Ext   100000    1          33             8
+Ethernet132         74,75     Ethernet34/1        34     Ext   100000    1          34             8
+Ethernet136         76,77     Ethernet35/1        35     Ext   100000    1          35             8
+Ethernet140         72,73     Ethernet36/1        36     Ext   100000    1          36             8
+Ethernet144         70,71     Ethernet37/1        37     Ext   100000    1          37             8
+Ethernet148         66,67     Ethernet38/1        38     Ext   100000    1          38             8
+Ethernet152         68,69     Ethernet39/1        39     Ext   100000    1          39             8
+Ethernet156         64,65     Ethernet40/1        40     Ext   100000    1          40             8
+Ethernet160         62,63     Ethernet41/1        41     Ext   100000    1          41             8
+Ethernet164         58,59     Ethernet42/1        42     Ext   100000    1          42             8
+Ethernet168         60,61     Ethernet43/1        43     Ext   100000    1          43             8
+Ethernet172         56,57     Ethernet44/1        44     Ext   100000    1          44             8
+Ethernet176         54,55     Ethernet45/1        45     Ext   100000    1          45             8
+Ethernet180         50,51     Ethernet46/1        46     Ext   100000    1          46             8
+Ethernet184         52,53     Ethernet47/1        47     Ext   100000    1          47             8
+Ethernet188         48,49     Ethernet48/1        48     Ext   100000    1          48             8
+Ethernet-Rec0       221       Recirc0/0           51     Rec   400000    0          221            8
```

Comment:
I think the Recycle port is also names as `Ethernet-IBx`. Can you check this ?

Reply:
The interface naming `Ethernet-IBx` is reserved for inband communication which is mandatory on a VOQ chassis.
The `Ethernet-RecX` is a recycle port that is currently targeted at `ERSPAN` and still optional.

Reply:
For reference, here is the hld https://github.com/Azure/SONiC/blob/master/doc/recirculation-port/recirculation_port.md